src/Spout/Writer/Common/Manager/Style/ManagedStyle.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Manager\Style;
+
+use Box\Spout\Common\Entity\Style\Style;
+
+class ManagedStyle
```

Comment:
I think it's worth documenting what the style is about (as the logic is not trivial to understand)

Reply:
Also, maybe rename that `PossiblyUpdatedStyle` or something along these lines.